Tasks/Common/VstsAzureHelpers_/InitializeAzModuleFunctions.ps1

```diff
@@ -6,15 +6,16 @@ function Initialize-AzModule {
     param(
         [Parameter(Mandatory=$true)]
         $Endpoint,
-        [string] $azVersion)
+        [string] $azVersion,
+        [string] $restrictContext = 'False')
```

Comment:
This is set to False since we are not providing checkbox option in V5 version of azurePowerShell, so this task will take this value to true if it is set to true which might break pipelines of customer who are using saved context, so I set it to false for now